src/api/unified/moments.cpp

```diff
@@ -0,0 +1,18 @@
+/*******************************************************
+ * Copyright (c) 2015, ArrayFire
+ * All rights reserved.
+ *
+ * This file is distributed under 3-clause BSD license.
+ * The complete license agreement can be obtained at:
+ * http://arrayfire.com/licenses/BSD-3-Clause
+ ********************************************************/
+
+#include <af/array.h>
+#include <af/moments.h>
+#include "symbol_manager.hpp"
+
+af_err af_moments(af_array* out, const af_array in, const af_moment_type moment)
+{
+    CHECK_ARRAYS(in);
+    return CALL(out, in, moment);
+}
```

Comment:
You should defined the moment function with the double return argument here too.